modules/test/playwright/tests/document-library-web/dmInstanceSettings.spec.ts

```diff
@@ -0,0 +1,47 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {expect, mergeTests} from '@playwright/test';
+
+import {isolatedSiteTest} from '../../fixtures/isolatedSiteTest';
+import {loginTest} from '../../fixtures/loginTest';
+import {documentLibraryPagesTest} from './fixtures/documentLibraryPagesTest';
+
+export const CheckOverwriteMaxSizetest = mergeTests(
+	documentLibraryPagesTest,
+	isolatedSiteTest,
+	loginTest()
+);
```

Comment:
All usages of `mergeTests` I'e seen assign the result to a variable with regular capitalization (e.g. `checkOverwriteMaxSizeTest`).  In any case, why not keep it simple and name it `test`?

Reply:
+1 to keep simply `test` any case `camelCase` as @adolfopa says

Reply:
Perfect, no problem, I'll change it to test. Thanks for the advice.

Reply:
Changed